MinPlatformPkg/PlatformInit/PlatformInitPei/PlatformInitPreMem.c

```diff
@@ -186,6 +187,29 @@ BuildMemoryTypeInformation (
                                &DataSize,
                                &MemoryData
                                );
+

```

Comment:
Can you please add an `EFI_ERROR()` check for the status from `PeiServicesLocatePpi()` and return (in addition to the assert)?

Reply:
Note: Had to add a comment about it here due to GitHub UI limitations around leaving comments outside direct code changes.

Reply:
Added.